.github/workflows/PackageValidation.yml

```diff
@@ -42,4 +42,4 @@ jobs:
       - name: Select Xcode Version
         run: sudo xcode-select --switch ${{ vars.TOOLPATH }}
       - name: Build and Test
-        run: xcodebuild test -scheme ${{ github.event.repository.name }} -destination '${{ matrix.platforms.destination }}' -skipMacroValidation
+        run: xcodebuild test -scheme swift-resource-provider -destination '${{ matrix.platforms.destination }}' -skipMacroValidation
```

Comment:
As this is becoming v1, may also want to add API breakage checks for PRs: https://github.com/mochidev/CodableDatastore/blob/main/.github/workflows/api.yml

Reply:
Oh nice didn't know about that. I'll add it to the default workflow.

Reply:
Might take a couple extra pushes to get it to work though.